en_US/advanced/cluster.md

```diff
@@ -341,3 +341,53 @@ also be allowed by the firewall. The port mapping rule is similar to the node di
 ports in `ekka mode`, but with the `BasePort = 5370`. That is, having
 `node.name = emqx@192.168.0.12` in `emqx.conf` should make the node listen on port `5370`,
 and port `5371` for `emqx1` (or `emqx-1`), and so on.
+
+## Asynchronous database replication
+
+::: danger
+This feature is highly experimental.
+You can enable it if you want to help us testing and tuning it.
+:::
+
+Some EMQ X features rely on the open-source [Mnesia](http://erlang.org/doc/apps/mnesia/Mnesia_overview.html) database.
+Mnesia uses full-mesh topology for transaction replication.
+While being blazing-fast in the small clusters, it has some scalability issues when the number of nodes in the cluster becomes large.
+Also it becomes more vulnerable to network splits.
+To address these issues, EMQ X supports an experimental extension to Mnesia called `RLOG`.
+This extension can be used in large EMQ X clusters (4 nodes and more).
+
+### RLOG principles
+
+RLOG extension works by separating nodes in the cluster into two logical groups:
+
+- core nodes
```

Comment:
describe the number of core nodes, required, suggested?

having only one core node can work?

Reply:
Done.